### PR TITLE
Desktop at UCL instruction improvements

### PR DESCRIPTION
**Pull request by alessandrofelder**
Addresses @dpshelio and @mattgillucl 's comments on Slack! Thanks both!

### CODE REVIEW

File: _pages/setup/desktop.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ At [Desktop@UCL](https://my.desktop.ucl.ac.uk/) you can find Anaconda and a
     ```
     OLDHOME="${HOME}"  
     export HOME="/n/"  
-    cd $HOME [[ ! -f OLDHOME/.bash_profile ]] && cp $HOME/.bash_profile $OLDHOME 
+    cd $HOME
+    [[ ! -f OLDHOME/.bash_profile ]] && cp $HOME/.bash_profile $OLDHOME
```
Comment (mattagape) [inline _pages/setup/desktop.md:15]: I don't think the cp command matches the description provided.

**Review by mattagape — CHANGES_REQUESTED** [review]
Please check the cp command and the description are in sync. I thik one of them is wrong

File: _pages/setup/desktop.md [hunk 2]
```diff
@@ -11,7 +11,10 @@ At [Desktop@UCL](https://my.desktop.ucl.ac.uk/) you can find Anaconda and a
...
     OLDHOME="${HOME}"  
     export HOME="/n/"  
-    cd $HOME [[ ! -f OLDHOME/.bash_profile ]] && cp $HOME/.bash_profile $OLDHOME 
+    cd $HOME
+    [[ ! -f OLDHOME/.bash_profile ]] && cp $HOME/.bash_profile $OLDHOME 
     ```
 1. Open a new git bash terminal and run: `conda init bash` (this may say that the operation failed, but it's OK) 
 1. Open a new terminal again. It may show a bash error at the start, but everything should work fine. 
+
+This checks whether you had created a startup configuration in your home directory and copies to the new home on the n drive.
```
Comment (dpshelio) [inline _pages/setup/desktop.md]:
```suggestion
This checks whether you had created a startup configuration in your home directory before and if it doesn't exist, it copies it to the old home on the `c` drive.
```

I believe, however, that this explanation if given at all, should be either below the point where it happens or linked here with a footnote.
You could use `<sub>*</sub>` at the end of the step, and `<sub>*</sub>: explanation` at the bottom of the page.

**Review by dpshelio — CHANGES_REQUESTED** [review]

**Comment by alessandrofelder** [comment]
Latest changes should address your helpful comments now, @dpshelio and @mattgillucl ! Thank you!

**Review by mattagape — APPROVED** [review]
@alessandrofelder I think this is OK to merge now

**Pull request merged**